lisa/operating_system.py

```diff
@@ -763,6 +766,20 @@ def _get_package_information(self, package_name: str) -> VersionInfo:
         )
         return self._cache_and_return_version_info(package_name, version_info)
 
+    def add_azure_core_repo(self) -> None:
+        self.install_packages(["gnupg", "software-properties-common"])
+        # no azure-compatscanner package in azurecore-debian
+        # use azurecore instead
+        codename = "bionic"
```

Comment:
please add comments for why hardcode codename here :)

Reply:
Done